docs/docs/migrating-from-v1-to-v2.md

```diff
@@ -999,8 +995,7 @@ The signature for using createRemoteFileNode changed in v2, it now expects a new
 
 ### Only allow defined keys on the node internal object
 
-The node internal object isn't meant for adding node data. Those should be added to the top-level object. We
-didn't document this in v1 nor validate against it but are now for v2.
+The node internal object isn't meant for adding node data. Those should be added to the top-level object. We now validate against it for v2.
```

Comment:
I think this change is less clear than the previous version, as it excludes some information. What about something like this?

```suggestion
The node `internal` object isn't meant for adding node data. While Gatsby v1 allows this behaviour we now validate against it for v2. Node data should be added as fields on the top-level node object.

[Check the Node interface docs](https://www.gatsbyjs.org/docs/node-interface/) for allowed fields.
```

Reply:
I like your fix a lot better. Thanks!